### ISSUE TITLE
load pretrained weights

### ISSUE THREAD

**Issue by ydhongHIT** [noahzn/Lite-Mono#153]
Hi, why is k starting with 'norm' excluded when importing pre-trained weights? https://github.com/noahzn/Lite-Mono/blob/4874b35df8ed4da16159ce8be8c697028b72bf76/trainer.py#L661

**Comment by noahzn** [comment]
Hi, because the pre-trained model was trained based on a classification task on ImageNet. For depth estimation we should not use some of the layers.

**Comment by ydhongHIT** [comment]
I'm not sure if I understand this correctly, but wouldn't this exclude the normalization layer as well?

**Comment by ydhongHIT** [comment]
By the way, is drop path rate the same for all models? I try to train lite-mono-tiny with drop path rate 0.2 but it didn't converge well.

**Comment by noahzn** [comment]
> I'm not sure if I understand this correctly, but wouldn't this exclude the normalization layer as well?

Hi, you can print out the network structures and see which layers are not loaded. I can't remember exactly.

**Comment by noahzn** [comment]
> By the way, is drop path rate the same for all models? I try to train lite-mono-tiny with drop path rate 0.2 but it didn't converge well.

As the training seed is not fixed, it is possible that the same drop rate will result in poor (or good) training. If you have problems with overfitting, you can try a higher drop rate. For the tiny version, 0.2 should be sufficient. For the evaluation, please evaluate on the last 10-15 epochs, as you might get the best model at an earlier epoch.

**Comment by GLtangzero** [comment]
@noahzn 
Hello:
I just check the paper and code, is that meaning the pretrain only used for post-net?
Because I'm not found any gradient load for the depth-encoder and decoder.

**Comment by noahzn** [comment]
@GLtangzero Hi, no, the ImageNet pre-trained weights are used in the depth-encoder, the pose-net uses resnet weights.

**Comment by GLtangzero** [comment]
@noahzn 
Hey, thanks for the super speed return!
I just find my mistake and delete the wrong commit.
Thanks again!

**Comment by noahzn** [comment]
> @noahzn Hey, thanks for the super speed return! I just find my mistake and delete the wrong commit. Thanks again!

You are welcome!

**Comment by noahzn** [comment]
I am now closing this issue as there is no further response.
